### DISCUSSION TITLE
neovim的xmake插件

### DISCUSSION THREAD

**Discussion by Mythos-404** [xmake-io/xmake#4090] [Show and tell]
仓库链接 [xmake.nvim](https://github.com/Mythos-404/xmake.nvim)

可能会有Bug欢迎提issues

<details> <summary>一些预览</summary>

![image](https://github.com/xmake-io/xmake/assets/96325829/7b818c4b-a6fe-42ad-8e9f-b3efd593eea3)
![image](https://github.com/xmake-io/xmake/assets/96325829/c094831c-13b9-4fab-bacd-33cf0722a5d2)
![image](https://github.com/xmake-io/xmake/assets/96325829/7bdba525-65de-4155-9206-cf8ef7cfb383)
![image](https://github.com/xmake-io/xmake/assets/96325829/03636152-55f1-4649-8914-c02e75624855)

</details>

**Comment by waruqi** [comment]
👍

**Reply by Mythos-404** [reply]
请问能给我的插件添加在xmake官方网站的外部插件列表里吗？😁

**Reply by waruqi** [reply]
你可以提 pr 过来
